mwclient/client.py

```diff
@@ -364,7 +365,7 @@ def raw_call(self, script, data, files=None, retry_on_error=True, http_method='P
             headers['Accept-Encoding'] = 'gzip'
         sleeper = self.sleepers.make((script, data))
 
-        scheme = 'https'
+        scheme = self.scheme
         host = self.host
         if isinstance(host, (list, tuple)):
             scheme, host = host
```

Comment:
Please deprecate this code path, as described [here](https://github.com/mwclient/mwclient/issues/212#issuecomment-453120269) and [here](https://github.com/mwclient/mwclient/issues/212#issuecomment-456792297).

@danmichaelo, would something like [this](https://github.com/mwclient/mwclient/blob/4157e147e07f12219a6b25e976ea87c8d5deff56/mwclient/client.py#L632-L637) be appropriate for the deprecation? Or would we want something supporting Python 3?

Reply:
@waldyrious Good idea! I guess DeprecationWarning is still the way to go even though it's hidden by default.

Reply:
Done